sdk/core/azure-core/inc/azure/core/http/curl_transport.hpp

```diff
@@ -13,8 +13,12 @@
 #include "azure/core/nullable.hpp"
 
 #include <chrono>
+#include <ctime>
 #include <memory>
 #include <string>
+namespace Azure { namespace Core { namespace Test {
```

Comment:
Put this in AZURE_ENABLE_TESTING or whatever it's called these days?

Reply:
done